src/include/txn.h

```diff
@@ -240,6 +244,10 @@ struct __wt_txn_op {
 			} mode;
 		} truncate_row;
 	} u;
+
```

Comment:
I suggest moving the auto-generated flag definition above into here, under the `HAVE_DIAGNOSTIC` ifdef, next to the flags definition. Most flags have the auto-list immediately preceding the item in the structure. You can look at `log.h` or `btree.h` or other headers for examples.

I also suggest renaming it to `WT_TXN_OP_REPEATED`.